xfuser/model_executor/pipelines/pipeline_flux.py

```diff
@@ -309,19 +310,30 @@ def __call__(
                     sync_only=True,
                 )
 
+        def vae_decode(latents):
+            latents = self._unpack_latents(
+                    latents, height, width, self.vae_scale_factor
+            )
+            latents = (
+                latents / self.vae.config.scaling_factor
+            ) + self.vae.config.shift_factor
+            
+            image = self.vae.decode(latents, return_dict=False)[0]
+            return image
+        
+        if not output_type == "latent":
+            if get_runtime_state().runtime_config.use_parallel_vae:
+                latents = get_world_group().broadcast_latent(latents,get_runtime_state().runtime_config.dtype)
```

Comment:
It's not very rigorous. If we enable dp/cfg，we can not broadcast like this.

Reply:
In the following commit, fix this issue.